src/controllers/content.controller.ts

```
@@ -884,14 +885,53 @@
+            queryData.graphemesMappedObj,
+            queryData.level_competency
+          ).then((contentData)=>{
+            contentArr = contentData['wordsArr'];
+          });
+        }
+
+
+        if(queryData.mechanics_id !== undefined){
+          contentArr.map((content) => {
+            const { mechanics_data } = content;
+            if(mechanics_data){
```

ESLint

[error] Replace `(mechanics_data)` with `·(mechanics_data)·`

(prettier/prettier)